dts/x86/intel/elkhart_lake.dtsi

```diff
@@ -689,11 +689,15 @@
 			status = "okay";
 		};
 
-		counter: counter@70 {
+		rtc: counter: rtc@70 {
```

Comment:
Since you're making this change, would it make sense to add `supported: rtc` to the board definition of ehl_crb too (as well as an alias to the board dts)? Same for the `ia32` change and any boards that use that? It can of course be done as a follow-up PR as well, but I feel like it'd be good to be consistent with this, i.e. if the dts has an rtc node label, then boards using it should declare `supported: rtc`.

Reply:
Sure, will add these changes.

Reply:
I think by adding supported: rtc for ia32 I am enabling rtc tests on qemu_x86 which will fail. I should remove supported: rtc or instead remove rtc/counter instance from ia32.dtsi file as it doesn't work in qemu anyway. Please provide your suggestions. @jhedberg @bjarki-trackunit

Reply:
You should remove `supported: rtc` from the `qemu_x86.yaml` file :) the `x86/acrn` board also uses the `ia32.dtsi`. I don't know it it uses it either, but I think it is safest to keep the rtc/counter node in the `ia32.dtsi` file.